release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1201,6 +1201,132 @@ For more information about the unsupported, community-maintained, version of the
+--
++
+(link:https://issues.redhat.com/browse/OCPBUGS-44126[OCPBUGS-44126])
+
+* Previously, some permissions required for linked actions were missing. 
+Linked actions create the subresources necessary for other {azure-short} resources that the Machine API and {product-title} require. 
+With this release, the Machine API provider for {azure-short} has the following additional permissions:
++
+--
+** `Microsoft.Compute/disks/beginGetAccess/action`
+** `Microsoft.KeyVault/vaults/deploy/action`
+** `Microsoft.ManagedIdentity/userAssignedIdentities/assign/action`
+** `Microsoft.Network/applicationGateways/backendAddressPools/join/action`
+** `Microsoft.Network/applicationSecurityGroups/joinIpConfiguration/action`
+** `Microsoft.Network/applicationSecurityGroups/joinNetworkSecurityRule/action`
+** `Microsoft.Network/ddosProtectionPlans/join/action`
+** `Microsoft.Network/gatewayLoadBalancerAliases/join/action`
+** `Microsoft.Network/loadBalancers/backendAddressPools/join/action`
+** `Microsoft.Network/loadBalancers/frontendIPConfigurations/join/action`
+** `Microsoft.Network/loadBalancers/inboundNatPools/join/action`
+** `Microsoft.Network/loadBalancers/inboundNatRules/join/action`
+** `Microsoft.Network/networkInterfaces/join/action`
+** `Microsoft.Network/networkSecurityGroups/join/action`
+** `Microsoft.Network/publicIPAddresses/join/action`
+** `Microsoft.Network/publicIPPrefixes/join/action`
+** `Microsoft.Network/virtualNetworks/subnets/join/action`
+--
++
+(link:https://issues.redhat.com/browse/OCPBUGS-44130[OCPBUGS-44130])
+
+* Previously, installing an {aws-short} cluster failed in certain environments on existing subnets when the `publicIp` parameter in the compute machine set custom resource was set to `false`. 
+With this release, a fix ensures that a configuration value set for `publicIp` no longer causes issues when the installation program provisions machines for your {aws-short} cluster in certain environment.  
+(link:https://issues.redhat.com/browse/OCPBUGS-44373[OCPBUGS-44373])
+
+* Previously, {gcp-short} clusters that used non-UEFI disks failed to load. 
+This release adds a check to ensure that disks are UEFI-compatible before enabling features that require UEFI, such as secure boot. 
+As a result, you can use non-UEFI disks if needed. 
+This change adds new permissions requirements for `compute.images.get` and `compute.images.getFromFamily`. (link:https://issues.redhat.com/browse/OCPBUGS-44671[OCPBUGS-44671])
+
+* Previously, when the {aws-short} DHCP option set was configured to use a custom domain name that contains a trailing period (`.`), {product-title} installation failed. 
+With this release, the logic that extracts the hostname of EC2 instances and turns them into kubelet node names trims trailing periods so that the resulting Kubernetes object name is valid. 
+Trailing periods in the DHCP option set no longer cause installation to fail. (link:https://issues.redhat.com/browse/OCPBUGS-45306[OCPBUGS-45306])
```

Comment:
```suggestion
Trailing periods in the DHCP network protocol set no longer cause installation to fail. (link:https://issues.redhat.com/browse/OCPBUGS-45306[OCPBUGS-45306])
```
just have never seen DHCP referred to as an "option set" before; seemed odd and non-specific